packages/app/src/cli/models/app/app.ts

```diff
@@ -95,13 +98,17 @@ export class App implements AppInterface {
     configurationPath: string,
     nodeDependencies: {[key: string]: string},
     webs: Web[],
-    ui: UIExtension[],
-    theme: ThemeExtension[],
-    functions: FunctionExtension[],
+    extensions: ExtensionInstance[],
     usesWorkspaces: boolean,
     dotenv?: DotEnvFile,
     errors?: AppErrors,
   ) {
+    // Temporary workaround while we migrate to use appModule features.
+    const functionsExt = extensions.filter((extension) => extension.features.includes('function'))
+    const themes = extensions.filter((extension) => extension.features.includes('theme'))
+    const uis = extensions.filter(
+      (extension) => !extension.features.includes('function') && !extension.features.includes('theme'),
```

Comment:
these lines make me think that maybe it'd be nice to have a `ui` feature, so that in the future we can add new extension types which are not ui, nor theme, nor function.

Reply:
Oh, there is a `ui` feature, but this is just a workaround to dev/deploy keep working.
for instance web_pixel doesn't have the ui feature, but until the refactor is complete we need to consider it a ui extension still

Reply:
App won't expose an object with function/theme/ui in the future